### DISCUSSION TITLE
Change height in fullscreen card

### DISCUSSION THREAD

**Discussion by FireStarter75** [dwainscheeren/dwains-lovelace-dashboard#739] [Q&A]
Is is possible to change the height between the options in a fullscreen card? (like 
![Screenshot_20231018-165034~2](https://github.com/dwainscheeren/dwains-lovelace-dashboard/assets/19826305/789b880b-f6e0-4390-b235-09e3ccf67d0f)
this thermostat card)

**Comment by dwainscheeren** [comment]
Hi,

I am closing the old Dwains Dashboard 3.x discussions as part of the DD3 cleanup.

DD3 is now in maintenance/community mode and is slowly being phased out. Active development is focused on Dwains Dashboard Next, the newest dashboard project:

https://github.com/dwainscheeren/dwains-dashboard-next

For chat, community help and discussion around DD3/DD Next, please join the Discord:

https://discord.gg/7yt64uX

If you still have a reproducible DD3 bug on the latest 3.9.x release, please open a new issue with current details. For new feature ideas, please try Dwains Dashboard Next.
